js/templates/modals/editListing/editListing.html

```diff
@@ -153,16 +154,44 @@ <h1 class="tx3 tx500 floL"><%= ob.polyT('editListing.sectionNames.photos') %></h
           </ul>
         </section>
         <hr class="dashed margR margL" />
-        <section class="js-scrollToSection" style="height: 350px;background-color: orange">
+        <section class="js-scrollToSection pad tx3">
+          <div class="overflowAuto">
+            <h1 class="tx3 tx500 floL"><%= ob.polyT('editListing.sectionNames.attachments') %></h1>
```

Comment:
Can we cross that bridge when we come to it? Mike is updating the designs and I'd rather wait for that. In either case, I think we'll need a PR to retrofit existing areas that use the current font.